README.md

```diff
@@ -56,7 +56,7 @@ Before you install and use the plug-in:
 
     For more information, see [Installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html).
 
-    **Note:** If you're using npm@7, an Internet connection is required during installation of the Secure Credential Store plug-in. Additionally, there is a prerequisite on Windows for npm@7 users: `npm install -g prebuild-install`
+    **Note:** If you're using npm@7 on Windows, there is [an NPM bug]((https://github.com/npm/cli/issues/2659)) that requires you to install a prerequisite first: `npm install -g prebuild-install`
```

Comment:
This also has to be updated in Zowe docs (and possibly other places) 😋

Reply:
I think we're already covered here because the Zowe docs link to the SCS readme 😛
https://docs.zowe.org/stable/user-guide/cli-swreqplugins.html

Reply:
I thought I saw this same statement in a PR to zowe docs before.
@MichaelABroadcom could you confirm?
Statement:
``` 
**Note:** If you're using npm@7, an Internet connection is required during installation of the 
Secure Credential Store plug-in. Additionally, there is a prerequisite on Windows for npm@7 
users: `npm install -g prebuild-install`
```

Reply:
I couldn't find it. sorry 😋